src/content/en/updates/2019/07/chrome-75-media-updates.md

```diff
@@ -0,0 +1,175 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A round up of the audio/video updates in Chrome 75: predicting whether playback will be smooth and power efficient for encrypted media and support of the video element's "playsInline" attribute hint.
+
+{# wf_updated_on: 2019-07-24 #}
+{# wf_published_on: 2019-07-22 #}
+{# wf_tags: news,chrome75,media #}
+{# wf_featured_image: /web/updates/images/generic/play-outline.png #}
+{# wf_featured_snippet: A round up of the audio/video updates in Chrome 75: predicting whether playback will be smooth and power efficient for encrypted media and support of the video element's "playsInline" attribute hint. #}
+{# wf_blink_components: Blink>Media #}
+
+# Audio/Video Updates in Chrome 75 {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+Chrome's media capabilities were updated in version 75. In this article, I'll
+discuss those new features which include:
+
+- Predicting whether playback will be smooth and power efficient for encrypted
+  media.
+- Support of the video element's `playsInline` attribute hint.
+
+## Encrypted Media: Decoding Info API  {: #encrypted-media}
+
+Since Chrome 66, web developers have been able to use the [Decoding Info API] to
```

Comment:
Decoding Info API
Don't use the form of a specification title on something that's not an actual specification title. This is actually part of Media Capabilities API. Naming it like this hides that. Please fix throughout.

Reply:
Done.